go/beanstalkd/beanstalkd.go

```diff
@@ -23,9 +23,16 @@ type ProgressMessage struct {
 	Event        string       `json:"event"`
 }
 
+type StringMessage struct {
```

Comment:
We report progress about filtering back to the frontend through string messages because "progress" is expected to be monotonically increasing, and not well suited for jobs that are really composed of multiple stages. So our current solution is to send log messages that are stored in the db and shown in the browser.